redash/cli/database.py

```diff
@@ -2,13 +2,15 @@
 
 manager = Manager(help="Manage the database (create/drop tables).")
 
+
 @manager.command
 def create_tables():
     """Create the database tables."""
-    from redash.models import create_db, init_db
-
-    create_db(True, False)
+    from redash.models import db, create_db, init_db
+    create_db(True, True)
     init_db()
+    db.session.commit()
```

Comment:
Why do you need the `db.session.commit()`? 

I assume `create_db` or `init_db` calls Flask-SQLAlchemy's `create_all()`, which will implicitly call a commit:
http://stackoverflow.com/questions/34410091/flask-sqlalchemy-how-can-i-call-db-create-all-and-db-drop-all-without-trigg